newIDE/app/src/Export/GenericExporters/OnlineWebExport.js

```diff
@@ -2,13 +2,86 @@
 import { Trans } from '@lingui/macro';
 import * as React from 'react';
 import Text from '../../UI/Text';
+import { Column, Line } from '../../UI/Grid';
+import TextField from '../../UI/TextField';
+import {
+  getBuildArtifactUrl,
+  type Build,
+} from '../../Utils/GDevelopServices/Build';
+import RaisedButton from '../../UI/RaisedButton';
+import Window from '../../Utils/Window';
+import Paste from '../../UI/CustomSvgIcons/Paste';
+import InfoBar from '../../UI/Messages/InfoBar';
 
 export const ExplanationHeader = () => (
-  <Text>
+  <Column noMargin alignItems="center" justifyContent="center">
+    <Line>
+      <Text align="center">
+        <Trans>
+          Generate a unique link to share your game, for a few days, playable
+          from any computer browser or mobile phone.
+        </Trans>
+      </Text>
+    </Line>
+  </Column>
+);
+
+type WebProjectLinkProps = {|
+  build: ?Build,
+  loading: boolean,
+|};
+
+export const WebProjectLink = ({ build, loading }: WebProjectLinkProps) => {
+  const [showCopiedInfoBar, setShowCopiedInfoBar] = React.useState<boolean>(
+    false
+  );
+
+  if (!build && !loading) return null;
+  const buildPending = loading || (build && build.status !== 'complete');
+
+  const value = buildPending
+    ? 'Just a few seconds while we generate the link...'
+    : getBuildArtifactUrl(build, 's3Key') || '';
+
+  const onOpen = () => {
+    if (buildPending) return;
+    Window.openExternalURL(value);
+  };
+
+  const onCopy = () => {
+    if (buildPending) return;
+    navigator.clipboard.writeText(value);
```

Comment:
I've never used it but we seem to have a Clipboard class for that

Reply:
Good observation. Speaking of which, it should probably be adapted to use the navigator.clipboard APIs now that they are implemented everywhere :) 
Note that the existing Clipboard is unfortunately limited to objects. We could rework it to also expose simple read/write texts apis.

Reply:
(we can add a TODO)

Reply:
Btw the clipboard class needs to be updated to use the web clipboard API if it can, right now it uses either the electron API or stores and read from a javascript variable if not in electron, making the contents of the clipboard not pasteable somewhere else on the web editor.